hack/kind-cluster-v1.12.8.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
+
+k8s_version=v1.12.8
+clusterName=${1:-kind}
+
+echo "############ install kind ##############"
+if hash kind 2>/dev/null;then
+    echo "kind have installed"
+else
+    echo "start install kind"
+    GO111MODULE="on" go get sigs.k8s.io/kind@v0.4.0
+    echo PATH=\$PATH:$(go env GOPATH)/bin >> /etc/profile && source /etc/profile
+fi
+
+echo "############# start create cluster:[${clusterName}] #############"
+
+data_dir=${HOME}/kind/${clusterName}/data
+
+echo "clean data dir: ${data_dir}"
+if [ -d ${data_dir} ]; then
+    rm -rf ${data_dir}
+fi
+
+echo "init the mount dirs for kind cluster"
+mkdir -p ${data_dir}/worker{,1,2,3,4,5}/vol{1..9}
+
+cat <<EOF > kind-config.yaml
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+networking:
+  disableDefaultCNI: true
+nodes:
+- role: control-plane
+- role: worker
```

Comment:
The default 6 worker nodes make it almost impossible to create a kind cluster on a laptop.

Reply:
I have tested the script on my laptop, it is no problem to create a cluster with 6 nodes. The appropriate node count is dependent by laptop resource, to make the node count custom may be better?
In the other hands, it is necessary to test the kind cluster's stability on the laptop.

Reply:
The following has too many duplicate codes. Could you use a better way to implement this? And also it would be better to allow customization of the kind cluster, like k8s version, worker count, volume count.